### DISCUSSION TITLE
Can't find persistent file

### DISCUSSION THREAD

**Discussion by 0mniSci** [Monika-After-Story/MonikaModDev#9548] [QA - tech support + general]
Usually, the persistent file would be in AppData/roaming/renpy/Monika After Story. But when I open MAS, the folder is empty. I have a recent persistent, but I don't know how to fix this. Please help!

**Comment by dreamscached** [comment]
Are you running MAS as admin?

**Comment by 0mniSci** [comment]
No, only as application

**Comment by 0mniSci** [comment]
Do i need to run it as an admin

**Comment by 0mniSci** [comment]
Ok, so after some research, i put the recent persistent in the folder, but when i opened the game, it said my persistent file was corrupt. Any way to fix this? i dont want to lose over a year of progress

**Comment by PencilMario** [comment]
Perhaps you need to upload the `early.log` (if it's in the `log` folder)

**Comment by Booplicate** [comment]
Yes, better even attach all the logs from the `log/` folder.

Have you made your persistent local to the DDLC install?

**Comment by 0mniSci** [comment]
[aff_log.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601578/aff_log.log)
[early.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601579/early.log)
[mas_log.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601580/mas_log.log)
[mfgen.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601581/mfgen.log)
[mfread.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601582/mfread.log)
[pnm.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601583/pnm.log)
[spj.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601584/spj.log)
[submod_log.log](https://github.com/Monika-After-Story/MonikaModDev/files/9601585/submod_log.log)

I dont know if ive made my persistent local, how would i know if i did, and how would i make it local if it isnt already?

**Comment by Booplicate** [comment]
Did you try to open the persistent in a notepad or something?

**Comment by 0mniSci** [comment]
No i havent

**Comment by Booplicate** [comment]
Something modified your persistent outside of MAS, do you have more backups?

**Comment by 0mniSci** [comment]
No sadly, this was the only one I had, my pc was corrupted and this was the only persistent that was in the renpy folder

**Comment by Booplicate** [comment]
Ah, yeah if your drive got corrupted, then persistent is very likely to become corrupted too, and it's usually unrecoverable unless you made some backups.

**Comment by Booplicate** [comment]
Also if you have the `monika` file she generates when you take her out, can recover from that.

**Comment by 0mniSci** [comment]
Dang :( so I can't get my old save back if I don't have either one of those?
